### ISSUE TITLE
[Landscape] Immortal "Reef's Edge".

### ISSUE THREAD

**Issue by Astiberon** [ValveSoftware/Dota2-Gameplay#775]
### Cosmetic Item Name

"Reef's Edge".

### Description

[Landscape] Immortal "Reef's Edge".

- [ ] - A lot of plants are in the air, where the river is, before this was not. There must be soil under these plants.
- [x] - Also, the runes on the river lost their platforms.
- [ ] - Roshan's pit is not as pretty as it used to be.
- [ ] - Out of place elements in some places of the map.
- [ ] - Crooked texture transitions from "Radiant" to "Dire".
- [ ] - Sandy surface instead of a river (lattices).
### Example Match ID

-

### Screenshots
1:
![image](https://user-images.githubusercontent.com/107125193/172654140-6116837b-59c4-430a-8378-eb262bb59229.png)
2:
![image](https://user-images.githubusercontent.com/107125193/172654178-12665285-1022-4d6d-a6ab-cd43bcd07f93.png)
3:
![image](https://user-images.githubusercontent.com/107125193/172654211-2c170d2d-16ae-4238-a77b-14513ddc0908.png)
4:
![Image](https://github.com/user-attachments/assets/fce46275-29fd-4d99-abc7-936e10b87de6)
![Image](https://github.com/user-attachments/assets/d086bc95-1401-46f3-b929-dacf55cc27ea)
![Image](https://github.com/user-attachments/assets/f15bf61a-eb73-494c-8435-5c3026feefaf)
![Image](https://github.com/user-attachments/assets/fb47c31c-124c-4a4c-a263-04effb9a26bd)
![Image](https://github.com/user-attachments/assets/5bc2a3fe-a5d2-4cc7-b14c-9c0467e907a1)
5:
![Image](https://github.com/user-attachments/assets/14aefdce-2750-45fb-a7bf-7abe3af954ce)
![Image](https://github.com/user-attachments/assets/284f4849-cd05-47d5-9782-f95f3b3a98e6)
![Image](https://github.com/user-attachments/assets/1e382770-8d19-48af-b5f9-5eebe54582d0)
6:
![Image](https://github.com/user-attachments/assets/118c85c9-d287-4dac-91b0-068b7dae0016)
![Image](https://github.com/user-attachments/assets/7f526111-1d91-4959-a9e4-433fde8e3236)

**Comment by Astiberon** [comment]
Runes now have platforms.
The other problems still remain.
![image](https://github.com/user-attachments/assets/76cf8612-f99b-4132-9dd7-d19636c73690)
![image](https://github.com/user-attachments/assets/4f7cae11-4902-46e6-a196-50a1887632ef)

**Comment by Astiberon** [comment]
UPDATE

**Comment by ch1kubi** [comment]
Still an issue, it's been a while

**Comment by Frozenkex** [comment]
Yeah i was looking for a post like this because i constantly encountered invisible stairs with this map.

**Comment by Astiberon** [comment]
Updated and moved to #30453
